File: aoj/volume3/n0317/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int gole = sc.nextInt();
        int perBigJump = sc.nextInt();

        int bigJump = gole / perBigJump;
        int smallJump = gole % perBigJump;

        int jumpMin = bigJump + smallJump;

        System.out.println(jumpMin);

    }

}

File: aoj/volume3/n0317/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int gole, perBigJump;
    cin >> gole >> perBigJump;

    int bigJump = gole / perBigJump;
    int smallJump = gole % perBigJump;

    int jumpMin = bigJump + smallJump;

    cout << jumpMin << endl;

    return 0;

}
